runtime/lua/vim/lsp/semantic_tokens.lua

```diff
@@ -192,11 +194,13 @@ function STHighlighter:new(bufnr)
       if not highlighter then
         return true
       end
-      highlighter:on_change()
+      if M.is_enabled({ bufnr = buf }) then
+        highlighter:on_change()
+      end
     end,
     on_reload = function(_, buf)
       local highlighter = STHighlighter.active[buf]
-      if highlighter then
+      if highlighter and M.is_enabled({ bufnr = bufnr }) then
```

Comment:
since `send_request()` itself checks `is_enabled(bufnr)`, do we need to add checks _around_ calls to `send_request()`?